src/main/java/life/qbic/projectwizard/views/AdminView.java

```diff
@@ -167,8 +167,13 @@ public void buttonClick(ClickEvent event) {
                   "There seems to have been a problem while creating the space. Do the specified users already exist in openbis? If not, create them.",
                   NotificationType.ERROR);
             }
-            createSpace.setEnabled(true);
+
+          } else {
+            Styles.notification("Problem creating space",
+                    "This space already exists.",
+                    NotificationType.DEFAULT);
```

Comment:
NIT: you could also add the space name in this notification

Reply:
It's still shown in the UI in this case, so not necessary.

Reply:
Perfect 👍